pkg/apis/core/validation/shoot_test.go

```diff
@@ -5865,24 +5865,30 @@ var _ = Describe("Shoot Validation Tests", func() {
 				fldPath = field.NewPath("cloudProfile")
 			})
 
+			It("should not allow using no cloudProfile reference (should be filled by sync synced in Shoot strategy)", func() {
```

Comment:
```suggestion
			It("should not allow using no cloudProfile reference", func() {
```

Might be rather confusing/irrelevant for the test description.